data/blog/app-feedback-program.md

```diff
@@ -0,0 +1,37 @@
+---
+title: Electron App Feedback Program
+author: sofianguy
+date: '2018-10-04'
+---
+
+Electron is working on building a streamlined release process and having faster releases. To help with that, we started the App Feedback Program for large-scale Electron apps to test our beta releases and report app-specific issues to the Electron team. We use this program to help us prioritize work and get applications upgraded to the next stable release as soon as possible.
+
+## Who can join?
+Our criteria and expectations for apps joining this program include the following items:
+- Test your app during the beta period for 10,000+ user-hours.
+- Have a single point-person to speak for the app's Electron bugs, app blockers, and check-in weekly, either during our regularly scheduled online meeting, or asynchronously with @sofianguy
+- You are willing to share the following information listed in the next question.
+
+## What info does my Electron app have to share?
+- Total user-hours your app has been running with any beta release.
+- Version of Electron that your app is testing with (e.g., 3.0.0-beta.3)
+- Any bugs preventing your application from upgrading to the release line being beta tested
+
+## User-hours
+We expect participants to test at least 10,000 user-hours as a minimum threshold for Electron to feel more comfortable releasing stable versions.
+- 10 user-hours could be 10 people testing for one hour, or one person testing for 10 hours.
+- You can split the testing between beta releases, for example test for 5,000 user-hours on 3.0.0-beta.2 and then test for 5,000 user-hours on 3.0.0-beta.5. More is better, but we understand that some applications cannot test every beta release.
+- CI or QA hours do not count towards the total, however internal releases do count.
+
+## Why should my Electron app join?
+Your app's bugs will be prioritized on the core Electron team's radar. In return, this is helpful for Electron to see how it is doing and what work needs to be prioritized.
```

Comment:
I know we've run at this before but I still don't think we have it quite right. "prioritized" is a risky word here and IMO we want to be very careful about not overpromising.

I'm not 100% happy with this revision, but something more like:

> Your app's bugs will be tracked and be on the core Electron team's radar. Your feedback helps the Electron team to see how the new betas are doing and what work needs to be done.

Reply:
I agree with @ckerr's wording.